### ISSUE TITLE
authorships in Zenodo releases

### ISSUE THREAD

**Issue by BenjaSanchez** [SysBioChalmers/yeast-GEM#223]
### Description of the issue:

In our [releasing guidelines](https://github.com/SysBioChalmers/yeast-GEM/blob/master/.github/CONTRIBUTING.md#releasing-a-new-version) we don't have any specification on how to assign authorships for each new Zenodo release (so all of them so far have [the same](https://zenodo.org/search?page=1&size=20&q=conceptrecid:%221494182%22&sort=-publication_date&all_versions=True) authors). I can manually edit them to be more consistent, but some questions for discussion before:
* Should all contributors be included in all releases, or only contributors that committed to each specific release?
* What about contributors that did not commit? (e.g. PR reviewers, commenters in issues, supervisors, etc.)

@edkerk @Hao-Chalmers how are you handling this in RAVEN?

**I hereby confirm that I have:**
- [x] Checked that a similar issue does not exist already

**Comment by Hao-Chalmers** [comment]
This issue hasn't be addressed in RAVEN yet.

**Comment by edkerk** [comment]
As @Hao-Chalmers said, we haven't addressed this in RAVEN, we've only been considering authorship with releasing major updates (= new publication).

I'd argue that committers are not the only ones making a valuable contribution to new releases (see e.g. #220, PR reviews can be quite extensive). There is a risk that this encourages people to Approve a PR without thorough Review, just to be included in the authorlist, but this is just for the Zenodo releases, so the reward is probably too low for this to become a problem.

Commenters can be left out by default, but what about Issue-starters, who highlighted a bug but did not fix it themselves (so did not author a PR)?

I'd leave out supervisors by default, if they have an active role then they would co-author or review the PR and be included by that criteria. Make clear that this is just for the Zenodo releases, paper authorship can also be justified for providing the funding, general supervision etc.

**Comment by BenjaSanchez** [comment]
I corrected manually all authorships to include both committers and reviewers. I left out issue starters, as I think in many cases it doesn't make sense (often it's just someone noticing a bug), and it would mean more work for the admin (some users don't have their actual name as Github handle, and some might not give their consent to be considered authors). I also opened #233 to update the admin guidelines. Unless there are any other points to bring up, this can be closed after the next release.
